css/03-generic.css

```diff
@@ -143,6 +145,7 @@ body.home header.entry-header {
 
 .site-header.featured-image {
 	min-height: 70vh;
+	padding: 0;
```

Comment:
@zoracreates This will apply to large screens as well - was it meant to be only a mobiles style (the PR name says mobile, so just checking!)

Reply:
I think I added that originally to have the header images not take up the whole window.

Reply:
@laras126 the problem was in mobile, but this style works on larger screens too. 
@slambert I only changed the padding on the sides, which was coming from the parent theme.